### DISCUSSION TITLE
Unique github cloners

### DISCUSSION THREAD

**Discussion by GenCoder8** [community/community#21573] [Other Feature Feedback, Questions, & Ideas]
<p>Hi just a quick question from new github user.</p>
<p>when I see unique clones made in the traffic page does that mean other people have cloned my repository or is that my own doing?</p>
<p>thx</p>

**Comment by galaxyeden** [comment, marked as answer]
<p>Hi <a class="mention" href="/gencoder8">@GenCoder8</a> and welcome to GitHub! <code>Unique cloners</code> is the number of unique people (determined by username or IP address) who have cloned your repository.</p>
<p>One note, however, is that these cloners may not have been specifically interested in your repository - there are services making use of our API endpoints, for example <code>https://api.github.com/events</code>, to monitor public events on GitHub for search indexes/archives/etc.</p>

**Comment by GenCoder8** [comment]
<p>Ok thanks for the answer. I was just surprised that there are 24 Unique cloners in two weeks and I haven’t advertised the repository at all</p>

**Comment by galaxyeden** [comment]
<p>You’re very welcome. Hopefully automated traffic helps explain this (even if it’s perhaps disappointing for a new project!). Please let us know if you have any more questions, and welcome again!</p>
